crates/net/network/src/peers/manager.rs

```diff
@@ -262,7 +262,7 @@ impl PeersManager {
     /// be scheduled.
     pub(crate) fn on_incoming_session_established(&mut self, peer_id: PeerId, addr: SocketAddr) {
         // we only need to check the peer id here as the ip address will have been checked at
-        // on_inbound_pending_session. We also check if the peer is in the backoff list here.
+        // on_incoming_pending_session. We also check if the peer is in the backoff list here.
         if self.ban_list.is_banned_peer(&peer_id) {
```

Comment:
maybe out of scope, but don't we want to decrease pending before this check? cc @mattsse 
```rust
self.connection_info.decr_pending_in();
```

Reply:
Seems the banned peer's pending_in was not increased, so no need to decrease

https://github.com/paradigmxyz/reth/blob/7849d1b75639b256276a200d50bfe2c6f9b8d74e/crates/net/network/src/peers/manager.rs#L225-L235